docs(managed-postgres): announce July 31st credential redaction, require provider 3.21.0 (#6559)
ClickHouse/clickhouse-docs: docs/cloud/managed-postgres/openapi.md

Let's explore the lifecycle of a Postgres service.

:::warning[Credential redaction from July 31st]

> On **July 31st**, the API stops returning the `password` and `connectionString` properties in its responses. After that date they appear only in the response that creates a service and the response of a [password reset] without a supplied password. Capture credentials from the create response; a later `GET` will not return them. If you manage services with Terraform, upgrade the provider to **v3.21.0 or later** before July 31st (see the [Terraform reference](/cloud/managed-postgres/terraform)).

Reviewer: ```suggestion
:::warning[Credential redaction from July 31, 2026]
As of **July 31, 2026**, the API no longer returns the `password` and `connectionString` properties in its responses. They appear only in the [create)[#create) response and [password reset] response when the request contained now password. We recommend that you capture credentials from the [create)[#create) response. If you manage services with Terraform, upgrade the provider to **v3.21.0 or later** before July 31st (see the [Terraform reference](/cloud/managed-postgres/terraform)).
```

```

Now use this data to create a new instance; note that it requires the content
type header. Save the response, since it is the one place the credentials are

> returned:

Reviewer: ```suggestion
type header. Save the response in order to have access to the credentials it returns:
```

    | jq
```

The output will be similar to the JSON returned for creation (from July 31st,

> without the `password` and `connectionString` properties), but keep an eye

Reviewer: ```suggestion
The output will be similar to the JSON returned for creation (but without the `password` and `connectionString` properties after July 31, 2026), but keep an eye
```

"running"
```

Now you can use the `connectionString` property from the saved create response

> to connect, for example via [psql]:

Reviewer: ```suggestion
Now you can use the `connectionString` property saved from the create response
to connect, for example via [psql]:
```
